website/templates/devices.html

```diff
@@ -44,41 +44,29 @@ <h4 class="modal-title" id="myModalLabel">Pair New Device</h4>
     <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#myModal">
         Pair New Device
     </button>
-    <p>
-        Below is a list of all your current devices.
-    </p>
-    <ol>
-        <li>Kitchen Thermostat
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-remove"></span>
-                Delete
-            </a>
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-plus"></span>
-                Add to group
-            </a>
-        </li>
-        <li>Living Room Sensor
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-remove"></span>
-                Delete
-            </a>
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-plus"></span>
-                Add to group
-            </a>
-        </li>
-        <li>Bathroom Door/Window Sensor
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-remove"></span>
-                Delete
-            </a>
-            <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
-                <span class="glyphicon glyphicon-plus"></span>
-                Add to group
-            </a>
-        </li>
-    </ol>
+    {% if paireddevices %}
+        <p>
+            Below is a list of all your current devices.
+        </p>
+        <ol>
+            {% for devices in paireddevices %}
+                <li>{{ devices.text }}
+                    <a href="{{ url_for('.devices') }}" class="btn btn-xs btn-default">
```

Comment:
Add the device ID to the url_for so we can navigate to the correct device.
Also, for "remove", use another endpoint.